packages/xz_java.rb

```diff
@@ -0,0 +1,35 @@
+require 'package'
+
+class Xz_java < Package
+  description 'A complete implementation of XZ data de/compression in pure Java.'
+  homepage 'http://tukaani.org/xz/java.html'
+  version '1.8'
+  source_url 'https://tukaani.org/xz/xz-java-1.8.zip'
+  source_sha256 '86be97adf59d19c000c5bac8c77f2fa2f84fa082d608c7ad2d07a007dce76160'
+
+  binary_url ({
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/xz_java-1.8-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+     x86_64: 'c8059feaa552660439ef17bb64a44fc096d9555eb06be8748cebf8ea2a7916dc',
+  })
+
+  depends_on 'jdk8'
+  depends_on 'ant' => :build
+
+  def self.build
+    system "JAVA_HOME=#{CREW_PREFIX}/share/jdk8", "ant"
+    system "JAVA_HOME=#{CREW_PREFIX}/share/jdk8", "ant", "doc"
+    system "wget", "https://github.com/JL2210/bash-scripts/raw/f786f858faae18c80e8c284ad4fc9e25cd14be36/xz-java.in"
```

Comment:
Why not just call this xz-java without the .in extension?  Also, I would prefer you provide the source for this script in this package instead of on your personal Github account.  It makes it easier to review and change (ie. more open).  There are a few things about your script I think can be improved.  Generally, most compression programs use `-x` instead of `-d` to extract/decompress.  Also, you could add `--compress` and `--decompress` long switch options to your help section.

Reply:
I'm trying to be as `xz` compliant as possible. `xz` has the -d option to decompress. It does have the long options `--compress` and `--decompress`. The file is too large to include in this package file. It installs as `xz-java`. It just has to configure the script with the prefix first.

Reply:
I'm talking about the help section, not the parsing logic.